demos/demo_runner/iot_demo_freertos.c

```diff
@@ -55,7 +57,7 @@ static IotSemaphore_t demoNetworkSemaphore;
 /* Variable used to indicate the connected network. */
 static uint32_t demoConnectedNetwork = AWSIOT_NETWORK_TYPE_NONE;
 
-#if MQTT_DEMO_TYPE_ENABLED
+#if defined( MQTT_DEMO_TYPE_ENABLED )
```

Comment:
Corresponding ```endif``` comment.

```
#endif /* #if defined( MQTT_DEMO_TYPE_ENABLED ) */
```

Reply:
Yea I thought about it, but since uncrustify didn't update it as such I thought I would just leave it.